web/wp-content/themes/borealis-2021/src/js/scripts/location.js

```diff
@@ -5,6 +5,7 @@ function locationModal(locationContainer) {
     const imageModal = locationContainer.querySelector("#img-modal");
     const containerHTML = document.querySelector("html");
     const imageSlideShow = locationContainer.querySelector("#image-slideshow");
+    const nav = document.querySelector("#main-navigation");
```

Comment:
```suggestion
    const nav = document.getElementById("main-navigation");
```
Is faster than `querySelector` because it does not need to search the whole DOM.  Since you're querying on an ID this may be helpful